<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 194, 218, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcom_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>600</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SVN-Beast</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Eye Protector</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_app">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/icon.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SVN-Beast</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SVN-Beast</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,182,193)
</string>
    </property>
    <property name="text">
     <string>Creat New Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forgor_password">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;

</string>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
